api/inc/uvisor_exports.h

```diff
@@ -34,15 +34,15 @@
 #define asm __asm__
 #endif
 
-/* compiler attributes */
-#define UVISOR_FORCEINLINE __attribute__((always_inline))
-#define UVISOR_NOINLINE    __attribute__((noinline))
+/* Shared compiler attributes */
+#if defined(__ICCARM__)
+#define UVISOR_FORCEINLINE inline
+#else
+#define UVISOR_FORCEINLINE inline __attribute__((always_inline))
+#endif
 #define UVISOR_PACKED      __attribute__((packed))
 #define UVISOR_WEAK        __attribute__((weak))
-#define UVISOR_ALIAS(f)    __attribute__((weak, alias (#f)))
-#define UVISOR_LINKTO(f)   __attribute__((alias (#f)))
 #define UVISOR_NORETURN    __attribute__((noreturn))
-#define UVISOR_NAKED       __attribute__((naked))
 #define UVISOR_RAMFUNC     __attribute__ ((section (".ramfunc"), noinline))
```

Comment:
These attributes are invalid for IAR.  You'll need to define them inside __ICCARM__ as well